msi/GitProduct.wxs

```diff
@@ -264,6 +264,8 @@
   </Fragment>
 
   <Fragment>
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f \"/[#GfwGitConfig]\" http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
@dscho: Ah, so that's what [#531](https://github.com/git-for-windows/git/issues/531) is all about.

So, if I understand your comment correctly:
1. `%ProgramData%\Git\config` _should not_ be uninstalled when Git for Windows is uninstalled
    1. If that's true, then likewise, this file _should not_ be installed if it already exists (e.g. a prior installation of Git for Windows installed this file, or perhaps an installation of libgit2).

2. It would seem then, that if 1.1 above is true, then:
    1. If the configuration setting `http.sslCAInfo` already has a valid value, Git for Windows should _not_ set this configuration setting upon installation. This complicates installation in general (especially upgrades, repairs, and uninstalls), but there may be a solution for this.
    2. If the configuration setting does not have a value (or an invalid value?), then Git for Windows _should_ set this value.
   3. Upon uninstallation of Git for Windows, if the setting is the setting set by Git for Windows, the setting should be _unset_. In this way, all paths within the system-wide git configuration are valid.

Did I miss anything? Is there anything else I should be aware of?

Also, it's possible that I could split this particular commit out of this PR and associate it with [#531](https://github.com/git-for-windows/git/issues/531) on a separae branch for the MSI installer (once this PR is merged), as this really is a product installation issue and not so much a specific installer technology installation issue.